hw/ip/otbn/dv/otbnsim/test/simple/add/expected.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+x10 = 1
+x11 = 0xffffffff
+x2 = 2
+x3 = 3
+x4 = 2
+x5 = 0xfffffffe
```

Comment:
Can we sort the registers numerically? (x10/x11 at the end)

Reply:
Do you think that's important? The loader is "imperative" and applies the updates in order (see `get_reg_expected`) and I thought it might be more useful to allow them to be in the same order as the code, where `x10` and `x11` are sort of helper registers, set at the start. What do you think?

Reply:
Nah, not important. Either behavior is just fine.

Reply:
Ok, cool. Let's start with this and, if it turns out to be annoying, we can always change it.